codespell_lib/data/dictionary.txt

```diff
@@ -11206,6 +11206,8 @@ extesnion->extension
 extesnions->extensions
 extint->extinct, extant,
 extist->exist
+extit->exit
+extiting->exiting
```

Comment:
A few more bits to add:
```suggestion
extited->excited, exited,
extiting->exciting, exiting,
extits->exits, excites,
```

Reply:
Thanks, done in ff6ebe1fbb9eae34ac8d1734e7f5a43b61feb522